### PR TITLE
Use glamorous for folding

### PR DESCRIPTION
**Pull request by swaterkamp**
Contributes to #17

### CODE REVIEW

File: ng/src/web/components/folding/folding.js [hunk 1]
```diff
@@ -107,12 +116,16 @@ export const withFolding = (Component, defaults = {}) => {
...
-        style={new_style}
-        {...props}
-        onTransitionEnd={onFoldStepEnd}
-        onAnimationEnd={onFoldStepEnd}
-      />
+      <Div
+        {...new_style}
+      >
+        <Component
+          style={new_style}
```
Comment (bjoernricks) [inline ng/src/web/components/folding/folding.js]: This prop should be removed from Component. It's being applied to the Div already

File: ng/src/web/components/folding/folding.js [hunk 2]
```diff
@@ -107,12 +116,16 @@ export const withFolding = (Component, defaults = {}) => {
     }
 
     return (
-      <Component
-        style={new_style}
```
Comment (bjoernricks) [inline ng/src/web/components/folding/folding.js]: The variable should be renamed. new_style isn't a good name anymore.

File: ng/src/web/components/folding/folding.js [hunk 3]
```diff
@@ -107,12 +116,16 @@ export const withFolding = (Component, defaults = {}) => {
...
-        onAnimationEnd={onFoldStepEnd}
-      />
+      <Div
+        {...new_style}
+      >
+        <Component
+          style={new_style}
+          {...props}
+          onTransitionEnd={onFoldStepEnd}
+          onAnimationEnd={onFoldStepEnd}
```
Comment (bjoernricks) [inline ng/src/web/components/folding/folding.js]: Maybe both handlers can be added to the Div instead. But I am not sure about that. Maybe Component isn't generic and is always some defined component which requires both handlers.

Comment (swaterkamp) [reply ng/src/web/components/folding/folding.js]: I tried moving them into the Div and the last remaining instance, where folding is used, still functions fine.

**Pull request merged**